presto-main/src/main/java/com/facebook/presto/metadata/DiscoveryNodeManager.java

```diff
@@ -207,4 +208,10 @@ private static NodeVersion getNodeVersion(ServiceDescriptor descriptor)
         String nodeVersion = descriptor.getProperties().get("node_version");
         return nodeVersion == null ? null : new NodeVersion(nodeVersion);
     }
+
+    private static Long getNodeStartTime(ServiceDescriptor descriptor)
+    {
+        String startTime = descriptor.getProperties().get("start_time");
+        return startTime == null ? null : Long.valueOf(startTime);
```

Comment:
Can the value of `start_time` ever be `null`? In `getNodeVersion()` the logic assumes that `node_version` value can be null. So I assumed the same, but then if this can be `null`, start time in the spi `Node` class can also be `null`, so this reflected itself in the spi, is this OK?